doc/web-doc/README.md

```diff
@@ -0,0 +1,47 @@
+# Documentation framework for Theia Trace Extension
+
+## Installation
+
+Requirements: NodeJS on the same version as the one required for [Theia IDE][theia-prereq].
+
+```bash
+cd doc/web-doc
+yarn
+```
+
+> Executing `yarn` from the root of theia-trace-extension does not install the dependencies in doc/web-doc. The configuration for Lerna to support such feature may be added in the future.
+
+Run site
+
+```bash
+yarn dev
+```
+
+Or start the server and open the app in a new browser tab
+
+```bash
+yarn dev --open
+```
+
+The site will be running at [http://localhost:5173](http://localhost:5173).
```

Comment:
I'd make this url also a reference linked from EOF, alongside the others.

Reply:
Done.